src/documents/InfusionEventSystem.md

```diff
@@ -141,26 +203,19 @@ Traditional procedural APIs corresponding to all the above declarations exist. H
 The Fluid event system is operated by instances of an *event firer* which are created by a call to `fluid.event.makeEventFirer()`. It is recommended that users don't construct event firers by hand but instead rely on the framework's facilities for automatically constructing these given event blocks in [component options](ComponentConfigurationOptions.md). The signature of `fluid.event.makeEventFirer` is not stable and will be revised in the 2.0 release of Infusion:
```

Comment:
It did change, yes - together with the signature for addListener (in its later arguments)